src/api/cdc/openapi.io.yaml.tpl

```diff
@@ -0,0 +1,171 @@
+          description: Validazioni non superate
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RichiestaCartaErrata'
+        "404": 
+          description: Risorsa non esiste
+        "401":
+          description: Utente non autorizzato
+        "403":
+          description: Utente non loggato
+        "500":
+          description: Errore interno.
+      # JWT Token (RSA+256)
+      security:
+        - BearerAuth: []
+  
+
+components:
+  securitySchemes:
+    BearerAuth:
+      type: http
+      scheme: bearer
+  schemas:
+    StatoBeneficiario:
+      type: string
+      enum: 
+        - ATTVABILE
+        - INATTIVABILE 
+        - ATTIVO # carta richiesta e risposta ok da INPS
+        - VALUTAZIONE # carta richiesta ma non si conosce la risposta da INPS
+        - INATTIVO # carta richiesta e risposta negativa da INPS
+    EsitoRichiesta:
+      type: string
+      enum: 
+        - INIZIATIVA_TERMINATA
+        - ANNO_NON_AMMISSIBILE
+        - CIT_REGISTRATO
+    Anno: 
+      type: string 
+      pattern: '^\d{4}$'
+      description: Anno a quattro cifre
+      example: "2000"
+    AnnoIsee:
+      type: object
+      required:
+        - anno
+      properties: 
+        anno:
+          $ref: '#/components/schemas/Anno'
+        dataIsee:
+          type: string
+    RichiestaCartaErrataMotivo: #Il payload della POST non può essere processato
+      type: string
+      enum: 
+        - NO_INPUT
+        - ANNI_RIFERIMENTO_NON_FORNITI
+        - LISTA_ANNI_VUOTA
+        - FORMATO_ANNI_ERRATO
+    RichiestaCartaErrata:
+      type: object
+      required:
+        - annoRiferimento
+        - status
+      properties:
+        type:
+          type: string
+        annoRiferimento:
+          $ref: '#/components/schemas/Anno'
+        title:
+          type: string
+        status:
+          $ref: '#/components/schemas/RichiestaCartaErrataMotivo'
+        detail:
+          type: string
+        instance:
+          type: string
+        errorCode:
+          type: string
+    StatoBeneficiarioPerAnno:
+      type: object
+      required:
+        - statoBeneficiario
+        - annoRiferimento
+      properties:
+        statoBeneficiario:
+          $ref: '#/components/schemas/StatoBeneficiario'
+        annoRiferimento:
+          $ref: '#/components/schemas/Anno'
+    EsitoRichiestaPerAnno:
+      type: object
+      required:
+        - statoBeneficiario
+        - annoRiferimento
+      properties:
+        statoBeneficiario:
+          $ref: '#/components/schemas/EsitoRichiesta'
+        annoRiferimento:
+          $ref: '#/components/schemas/Anno'
+    ListaStatoPerAnno:
+      type: object
+      properties:
+        listaStatoPerAnno:
+          type: array
+          items:
+            $ref: '#/components/schemas/StatoBeneficiarioPerAnno'
+    ListaEsitoRichiestaPerAnno:
+      type: object
+      properties:
+        listaEsitoRichiestaPerAnno:
```

Comment:
The `listaEsitoRichiestaPerAnno` should be required?

Reply:
I agree. I'll fix it